### PR TITLE
Update UseCaseValidator

### PR DESCRIPTION
**Pull request by r7-kszczepanskagorna**
## Description
Update UseCaseValidator to print valid use cases
Update UseCaseValidator to identify duplicates use cases in keywords

## Testing
There was added unit test.

### CODE REVIEW

**Comment by jschipp-r7** [comment]
@r7-kszczepanskagorna there's a merge conflict here

**Comment by jschipp-r7** [comment]
@r7-kszczepanskagorna there's two file conflicts here yet

**Comment by r7-kszczepanskagorna** [comment]
@jonschipp ready :)

**Review by jschipp-r7 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ be on your way to contributing!
 
 ## Changelog
 
+* 2.31.0 - Update UseCaseValidator to print valid use cases | Update UseCaseValidator to identify duplicates use cases in keywords
```
Comment (mrinehart-r7) [inline README.md:65]:
```suggestion
* 2.31.0 - Update UseCaseValidator to print valid use cases | Update UseCaseValidator to identify duplicate use cases in keywords
```

**Review by mrinehart-r7 — DISMISSED** [review]

**Review by mrinehart-r7 — APPROVED** [review]

**Review by jschipp-r7 — APPROVED** [review]

**Pull request merged**